assets/robots.txt

```diff
@@ -4,5 +4,5 @@
 
 
 # To prevent search engines from seeing the site altogether, uncomment the next two lines:
-# User-Agent: *
-# Disallow: /
+User-Agent: *
+Disallow: /preview
```

Comment:
Should this actually be `/preview/` with a trailing slash? I had to hunt down some docs as I haven't bothered with robots.txt for ages. Can we test this?